package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "lerna run build",
+    "build": "lerna run build --ignore=website",
```

Comment:
Because we could only run `npm run typedoc` in root folder only after all the packages are built (instead of website). The `npm run build` command in `website/` would do things to Typedocs and put them in the `versioned_docs/v1.x/` folder. Which means the runnable order of the script should be 
1. build all packages
2. build typedoc and soliditydocs
3. build website

Of course we could ignore the `setup-typedoc` and `setup-soliditydocs` command for the first build while running `lerna run build` command, and build typedocs and soliditydocs, and build the website again; but I don't think we should build the website twice. That's why I ignore the website package for the first build.